src/main/java/com/alkemy/ong/controllers/OrganizationController.java

```diff
@@ -2,31 +2,42 @@
 
 import com.alkemy.ong.dto.request.organization.OrganizationRequestDto;
 import com.alkemy.ong.dto.response.Organization.OrganizationPublicDto;
+import com.alkemy.ong.dto.response.slide.SlideResponseDto;
 import com.alkemy.ong.services.OrganizationService;
+import com.alkemy.ong.services.SlideService;
 import lombok.AllArgsConstructor;
 import org.springframework.http.ResponseEntity;
 import org.springframework.web.bind.annotation.*;
 
+import java.util.List;
+
 @RestController
 @RequestMapping("/organization")
 @AllArgsConstructor
 public class OrganizationController {
 
     private final OrganizationService organizationService;
+    private final SlideService slideService;
 
-    @GetMapping("/public")
+/*    @GetMapping("/public")
     public ResponseEntity<OrganizationPublicDto> publicData(@RequestParam String id) {
         OrganizationPublicDto publicDataDTO = organizationService.getPublicOrganizationData(id);
         return ResponseEntity.ok().body(publicDataDTO);
-    }
+    }*/
+@GetMapping("/public")
```

Comment:
Creo que el endpoint deberia seguir devolviendo los datos de la organizacion, y ademas se deberia agregar el listado de slides a la respuesta.

Reply:
The interpretation is changed from displaying a list of slides with a common organization to displaying an organization with a list of slides.